Topic4/src/classes/WebUserServiceResult.java

```diff
@@ -0,0 +1,20 @@
+package classes;
+
+public class WebUserServiceResult {

```

Comment:
You could have setCode and setMessage methods in this class which would allow you to create, in WebUserService.class, a general message at the beginning of each method and later update them according the result (if needed)